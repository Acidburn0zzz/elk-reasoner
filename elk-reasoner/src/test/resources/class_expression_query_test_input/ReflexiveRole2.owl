Prefix(:=<owlapi:ontology#ont23#>)
Prefix(owl:=<http://www.w3.org/2002/07/owl#>)
Prefix(rdf:=<http://www.w3.org/1999/02/22-rdf-syntax-ns#>)
Prefix(xml:=<http://www.w3.org/XML/1998/namespace>)
Prefix(xsd:=<http://www.w3.org/2001/XMLSchema#>)
Prefix(rdfs:=<http://www.w3.org/2000/01/rdf-schema#>)


Ontology(<owlapi:ontology#ont23>

Declaration(Class(<http://example.org/A>))
Declaration(Class(<http://example.org/B>))
Declaration(Class(<http://example.org/C>))
Declaration(Class(<http://example.org/C1>))
Declaration(Class(<http://example.org/C2>))
Declaration(Class(<http://example.org/D>))
Declaration(Class(<http://example.org/F>))
Declaration(Class(<http://example.org/K1>))
Declaration(Class(<http://example.org/L>))
Declaration(ObjectProperty(<http://example.org/H>))
Declaration(ObjectProperty(<http://example.org/H1>))
Declaration(ObjectProperty(<http://example.org/R>))
Declaration(ObjectProperty(<http://example.org/S>))
Declaration(ObjectProperty(<http://example.org/T1>))
Declaration(ObjectProperty(<http://example.org/T2>))
Declaration(ObjectProperty(<http://example.org/U>))
Declaration(ObjectProperty(<http://example.org/V1>))
Declaration(ObjectProperty(<http://example.org/V2>))
Declaration(ObjectProperty(<http://example.org/V3>))
############################
#   Object Properties
############################

# Object Property: <http://example.org/H> (<http://example.org/H>)

SubObjectPropertyOf(<http://example.org/H> <http://example.org/H1>)
ReflexiveObjectProperty(<http://example.org/H>)

# Object Property: <http://example.org/R> (<http://example.org/R>)

ReflexiveObjectProperty(<http://example.org/R>)


############################
#   Classes
############################

# Class: <http://example.org/B> (<http://example.org/B>)

EquivalentClasses(<http://example.org/B> ObjectSomeValuesFrom(<http://example.org/R> <http://example.org/A>))

# Class: <http://example.org/C1> (<http://example.org/C1>)

SubClassOf(<http://example.org/C1> ObjectSomeValuesFrom(<http://example.org/T1> ObjectSomeValuesFrom(<http://example.org/T2> <http://example.org/C2>)))

# Class: <http://example.org/D> (<http://example.org/D>)

EquivalentClasses(<http://example.org/D> ObjectSomeValuesFrom(<http://example.org/S> <http://example.org/C>))

# Class: <http://example.org/L> (<http://example.org/L>)

EquivalentClasses(<http://example.org/L> ObjectSomeValuesFrom(<http://example.org/V3> <http://example.org/K1>))


SubClassOf(ObjectSomeValuesFrom(<http://example.org/U> <http://example.org/C2>) <http://example.org/F>)
SubObjectPropertyOf(ObjectPropertyChain(<http://example.org/R> <http://example.org/H1>) <http://example.org/S>)
SubObjectPropertyOf(ObjectPropertyChain(<http://example.org/R> <http://example.org/V1>) <http://example.org/V2>)
SubObjectPropertyOf(ObjectPropertyChain(<http://example.org/T1> <http://example.org/T2> <http://example.org/H>) <http://example.org/U>)
SubObjectPropertyOf(ObjectPropertyChain(<http://example.org/V2> <http://example.org/R>) <http://example.org/V3>)
)