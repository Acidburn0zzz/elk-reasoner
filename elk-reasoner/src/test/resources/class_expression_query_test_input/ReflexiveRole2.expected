Prefix(:=<owlapi:ontology#ont24#>)
Prefix(owl:=<http://www.w3.org/2002/07/owl#>)
Prefix(rdf:=<http://www.w3.org/1999/02/22-rdf-syntax-ns#>)
Prefix(xml:=<http://www.w3.org/XML/1998/namespace>)
Prefix(xsd:=<http://www.w3.org/2001/XMLSchema#>)
Prefix(rdfs:=<http://www.w3.org/2000/01/rdf-schema#>)


Ontology(<owlapi:ontology#ont24>

Declaration(Class(<http://example.org/K1>))
Declaration(Class(<http://example.org/L>))
Declaration(ObjectProperty(<http://example.org/V1>))

############################
#   Classes
############################

# Class: owl:Nothing (owl:Nothing)

SubClassOf(owl:Nothing ObjectSomeValuesFrom(<http://example.org/V1> <http://example.org/K1>))



SubClassOf(ObjectSomeValuesFrom(<http://example.org/V1> <http://example.org/K1>) <http://example.org/L>)
)